controllers/managedocs_controller.go

```diff
@@ -1228,8 +1228,16 @@ func (r *ManagedOCSReconciler) reconcileOCSCSV() error {
 		return fmt.Errorf("OCS CSV not found")
 	}
 	var isChanged bool
+	var _0 = int32(0)
```

Comment:
This has no use above the condition (see comment about the issue in the condition) so you can just move it inside the if